usr/bin/torbrowser

```diff
@@ -433,9 +464,10 @@ Then try again.</p>'
 
 main_function() {
    root_check "$@"
-   tb_preparation "$@"
    tb_set_links "$@"
    parse_cmd_options "$@"
+   tb_preparation "$@"
+   tb_config_folder_parser "$@"
```

Comment:
Why? Looks like a bug. It would mean, that parse_cmd_options weights less than tb_config_folder_parser. Should be:

least priority: defaults
medium priority: config
highest priority: command line / environment variables

Reply:
I cannot remember why I did that. I will investigate and fix it shortly.